pom.xml

```diff
@@ -96,8 +97,18 @@
   <!-- Apache project inception year for generating correct NOTICE file for jar bundle. -->
   <inceptionYear>2018</inceptionYear>
 
+  <distributionManagement>
+    <repository>
+      <id>bintray-linkedin-maven</id>
+      <name>linkedin-maven</name>
+      <url>https://api.bintray.com/maven/linkedin/maven/pinot/;publish=1</url>
+    </repository>
+  </distributionManagement>
+
   <properties>
     <pinot.root>${basedir}</pinot.root>
+    <revision>0.4.0</revision>
```

Comment:
In this case, what will be the behavior of command `mvn release` how is version been updated there?

Reply:
Since now the version consists of revision and sha1, you can set the version number and then publish the release. An instruction can be found in the link below:
https://techluminary.com/discard-maven-release-plugin-with-a-new-approach/